btrdb/stream.py

```diff
@@ -1626,8 +1630,12 @@ def latest(self):
             lambda s: s.nearest(start, version=versions.get(s.uuid, 0), backward=True),
             self._streams,
         )
-        for point, _ in latest_points_gen:
-            latest.append(point)
+        for point in latest_points_gen:
+            if point is not None:
+                latest.append(point[0])
+            else:
+                # add RawPoint with None as timestamp and value
+                latest.append(None)
```

Comment:
Maybe im misunderstanding, but it sounded like you wanted to use the rawpoint option? Or that is what the code suggests here, but it is just a None type?

Reply:
oh, forgot to remove the comment here. I do want to use the RawPoint option, but seems to make sense to just having a None added.
thoughts?

Reply:
Ah i gotcha, yeah that seems reasonable to me. Just wanted to confirm.